apis/Google.Cloud.PubSub.V1/docs/index.md

```diff
@@ -82,6 +82,139 @@ restart listening for messages with `StartAsync(...)` again. Due to the expense
 instance, it is recommended that a singleton client per topic is used for the lifetime of the
 application.
 
+## Dependency Injection
```

Comment:
Wondering outloud whether we sould add extension methods to configure a PublisherClient provider, so that publishers for different topics may be registered and fetched by the topic name. Same for subscribers.

This would just be a utility that users could built themselves of course.

@jskeet what do you think?

Reply:
I suggest we wait until people ask for it. It's easy enough to add later.